tests/functional/test_eclab_parser.py

```diff
@@ -0,0 +1,251 @@
+"""
+The module tests correct parsing of mpt files generated by EC-lab with use of Zilien.
+The dataset with the files exists as a submodule
+
+NOTE: Test of PEIS technique is skipped, because it contains two columns with
```

Comment:
I suggest a wording along the lines of "not supported along with normal U, I based measurements".